app/lib/features/space/settings/pages/visibility_accessibility_page.dart

```diff
@@ -249,7 +249,7 @@ class _VisibilityAccessibilityPageState
       final spaceId = await selectSpaceDrawer(context: context);
       if (spaceId != null) {
         final spaceList =
-            await ref.read(joinRulesAllowedRoomsProvider(spaceId).future);
+            await ref.read(joinRulesAllowedRoomsProvider(roomId).future);
         final isAlreadyAdded = spaceList.any((roomId) => roomId == spaceId);
```

Comment:
What is different between `roomId` and `widget.roomId`?

Reply:
the ```roomId``` is the id we selected from drawer while ```widget.roomId``` is the room identifier we've opened settings.

Reply:
When I look at entire code file, it looks like both are same. widget.roomId pass to the function argument where roomId is the param.